src/library/pagination/pagination.scss

```diff
@@ -0,0 +1,16 @@
+ngbs-pagination {
+  display: block;
+
+  > .pagination {
+    user-select: none;
+
+    // it would be helpful to have access to user's variables here... i'm not sure if that's possible though.
+    // it's not great that the user can't see the selected page when ngbs-pagination is disabled
+    // it's also not great to hard-code these color values
+    // perhaps we leave it as is, and the user can override the style if they choose
```

Comment:
I totally agree!!!

It's a real shame not to rely on the abundance of variable bs4 comes with. But to allow this, the build process will have to export a single sass file that can be imported by the library-user into their own sass build. This requires some redesigning...